src/main/java/AacademyCourse/demohomeworkmodule14/service/AddressService.java

```diff
@@ -0,0 +1,36 @@
+package AacademyCourse.demohomeworkmodule14.service;
+
+import AacademyCourse.demohomeworkmodule14.entity.Address;
+import AacademyCourse.demohomeworkmodule14.repository.AddressRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class AddressService {
+    @Autowired
```

Comment:
This @Autowired should be related to constructor.

Reply:
Conversation resolved